lib/uo_cli.rb

```diff
@@ -459,6 +463,12 @@ def self.create_project_folder(dir_name, empty_folder = false, overwrite_project
           scenario_report_features = reopt_post_processor.run_scenario_report_features(scenario_report: scenario_report, save_names_feature_reports: ['feature_optimization'] * scenario_report.feature_reports.length, save_name_scenario_report: 'feature_optimization')
           puts "\nDone\n"
         end
+      elsif @opthash.subopts[:visualize] == true
+        visualization = URBANopt::Scenario::ResultVisualization.create_visualization(@root_dir)
```

Comment:
Does this need to be set to a variable, or can `visualization = ` be removed? Katherine had a good suggestion about checking for tests passing locally, can you confirm that?

Reply:
@vtnate the variable was not necessary, I removed it.